app/views/candidate_interface/personal_details/immigration_route/_shared_form.html.erb

```diff
@@ -0,0 +1,27 @@
+<%= f.govuk_error_summary %>
+
+<%= f.govuk_radio_buttons_fieldset(
+  :immigration_route,
+  legend: { text: t('page_titles.immigration_route'), size: 'xl' },
+) do %>
+  <%= f.govuk_radio_button(
+    :immigration_route,
+    'visa_sponsored_by_provider',
+    label: { text: 'A visa sponsored by a course provider' },
+    link_errors: true,
+  ) %>
+  <%= f.govuk_radio_button(
+    :immigration_route,
+    'other_route',
+    label: { text: 'Another route' },
+    hint: { text: 'For example, if you are applying for a Family visa or a Graduate visa' },
+  ) do %>
+  <%= f.govuk_text_area(
+    :immigration_route_details,
+    label: { text: 'Give details' },
+    max_words: 200,
```

Comment:
Please add a `threshold` of 90% - this means that the word count message won't show up until they hit 90% of the word count, as hopefully it'll be very rare that people try to enter a lot of information here.

```suggestion
    max_words: 200,
    threshold: 90
```